pygmo/docstrings.cpp

```diff
@@ -1625,7 +1629,8 @@ Multi Objective Evolutionary Algorithms by Decomposition (the DE variant)
 
 Raises:
     OverflowError: if *gen*, *neighbours*, *seed* or *limit* are negative or greater than an implementation-defined value
-    ValueError: if *weight_generation* is not one of 'random', 'low discrepancy', 'grid'
+    ValueError: if *decomposition* is not one of 'tchebycheff', 'weighted' or 'bi'
```

Comment:
Changed. But I think there are cases and this was one where the list is as easily readable as enumerating them